src/os/ObjectStore.cc

```diff
@@ -146,8 +146,13 @@ ostream& operator<<(ostream& out, const ObjectStore::Sequencer& s)
   return out << "osr(" << s.get_name() << " " << &s << ")";
 }
 
+ostream& operator<<(ostream& out, const ObjectStore::Transaction& tx) {
+
+  return out << "Transaction address = " << &tx;
```

Comment:
note that the Transaction pointer will change every time it  gets moved, so this won't help us trace a given transaction all the way through the system